src/angularComponent/ngcOmniboxController.js

```diff
@@ -256,7 +256,7 @@ export default class NgcOmniboxController {
    * Un-highlights all suggestions.
    */
   highlightNone() {
-    if (this.isHighlightingDisabled) {
+    if (this.isHighlightingDisabled || this._requireMatch) {
```

Comment:
Let's test this somehow. I thought that a lot of other methods depend on calling `highlightNone()` to work.